docs/admin/upgrading_fedora.rst

```diff
@@ -1,4 +1,4 @@
-Upgrading to Fedora 35
+Upgrading to Fedora 36
```

Comment:
We can remove this section on 4.1.1, which ships with fedora-36. Unless 4.1.1 is released as this is being reviewed, I'd recommend merging this PR as-is, as it'll be useful to have this in the commit history in case we need to reinstate the section with the next version bump.

Reply:
Also referred to in #109, which will need to be revised (or updated post-merge) if this section is removed for Qubes 4.1.1.